src/main/java/com/gregtechceu/gtceu/client/ClientProxy.java

```diff
@@ -56,6 +59,15 @@ public static void onRegisterEntityRenderers(EntityRenderersEvent.RegisterRender
         event.registerBlockEntityRenderer(GTBlockEntities.GT_SIGN.get(), SignRenderer::new);
         event.registerBlockEntityRenderer(GTBlockEntities.GT_HANGING_SIGN.get(), HangingSignRenderer::new);
     }
+    @SubscribeEvent
+    public static void onRegisterItemDecorations(RegisterItemDecorationsEvent event) {
+        for (Item item : BuiltInRegistries.ITEM) {
+            if (item instanceof IGTTool || item instanceof IComponentItem) {
+                event.register(item, new GTItemBarRenderer());
```

Comment:
IMO this should use a static instance of GTItemBarRenderer, or is that not possible?

Reply:
Changed it to use one :+1: